packages/transmission-rpc/transmission-rpc.1.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "Benoit Bataille <benoit.bataille@gmail.com>"
+authors: [ "Benoit Bataille <benoit.bataille@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/bataille/ocaml-transmission-rpc"
+bug-reports: "https://github.com/bataille/ocaml-transmission-rpc/issues"
+dev-repo: "https://github.com/bataille/ocaml-transmission-rpc"
```

Comment:
This needs `.git` on the end so opam can work out which version control system to use.